.github/dependabot.yml

```diff
@@ -25,5 +25,3 @@ updates:
     schedule:
       interval: daily
     open-pull-requests-limit: 10
-    ignore:
-      - dependency-name: github.com/submariner-io/*
```

Comment:
Does this actually work on devel commits? What we want is bumps to the latest commit on the devel branches; I wonder if this won’t cause dependabot to submit PRs updating to the latest tag, either on devel or on the last release branch once it exists (which will do the wrong thing).